ui/src/pages/Dashboard/NodeInfo/NodeAddressInfo.vue

```diff
@@ -0,0 +1,283 @@
+                  <p><b>Total Nimbus:</b> {{ props.row.totalNimbus }}</p>
+                  <p><b>Total Stratus:</b> {{ props.row.totalStratus }}</p>
+                </template>
+              </el-table-column>
+              <el-table-column
+                v-for="column in tableColumns"
+                :key="column.label"
+                :min-width="column.minWidth"
+                :prop="column.prop"
+                :label="column.label"
+                sortable
+              />
+            </el-table>
+          </div>
+        </div>
+        <div
+          slot="footer"
+          class="col-12 d-flex justify-content-center justify-content-sm-between flex-wrap"
+        >
+          <div class="">
+            <p class="card-category">
+              Showing {{ from + 1 }} to {{ to }} of {{ total }} entries
+            </p>
+          </div>
+          <l-pagination
+            v-model="pagination.currentPage"
+            class="pagination-no-border"
+            :per-page="pagination.perPage"
+            :total="pagination.total"
+          />
+        </div>
+      </card>
+    </div>
+  </div>
+</template>
+<script>
+import {
+  Table, TableColumn, Select, Option,
+} from 'element-ui';
+import { Pagination as LPagination } from 'src/components/index';
+import Fuse from 'fuse.js';
+import axios from 'axios';
+import Loading from 'vue-loading-overlay';
+import 'vue-loading-overlay/dist/vue-loading.css';
+
+export default {
+  components: {
+    LPagination,
+    [Select.name]: Select,
+    [Option.name]: Option,
+    [Table.name]: Table,
+    [TableColumn.name]: TableColumn,
+    Loading,
+  },
+  data() {
+    return {
+      pagination: {
+        perPage: 100,
+        currentPage: 1,
+        perPageOptions: [5, 10, 25, 50, 100, 200, 500, 1000, 2000, 5000, 10000],
+        total: 0,
+      },
+      searchQuery: '',
+      propsToSearch: ['zelId'],
+      tableColumns: [
+        {
+          prop: 'zelId',
+          label: 'Zel ID',
+          minWidth: 70,
+        },
+        {
+          prop: 'paymentId',
+          label: 'Payment ID',
+          minWidth: 40,
+        },
+        {
+          prop: 'org',
+          label: 'Organization',
+          minWidth: 40,
+        },
+        {
+          prop: 'totalNodes',
+          label: 'Total Nodes',
+          minWidth: 90,
+        },
+      ],
+      tableData: [],
+      values: [],
+      fuseSearch: null,
+      isLoading: false,
+      paymentAddress: null,
+      organization: null,
+      totalNodes: null,
+      totalCumulus: null,
+      totalStratus: null,
+      totalNimbus: null,
+      zelids: [],
+    };
+  },
+  computed: {
+    pagedData() {
+      return this.tableData.slice(this.from, this.to);
+    },
+    /** *
+     * Searches through table data and returns a paginated array.
+     * Note that this should not be used for table with a lot of data as it might be slow!
```

Comment:
What is large dataset table for this scenario?
If we do filtering on server, then server would be handling the load and sending data - which can cause very high load on our infra.